index.html

```diff
@@ -1,9 +1,80 @@
-<!DOCTYPE html>
-<html lang="ru">
-<head>
-    <meta charset="utf-8">
-    <title>Задача «Подписка на бложик»</title>
-</head>
-<body>
-</body>
-</html>
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="utf-8"/>
+    <title>Задача "подписка на бложик"</title>
+</head>
+<body>
+    <table>
+        <tr>
+            <td>
+                <div id="subscribe" class="form">
+                    <form>
+                        <h2>Хочу подписаться!</h2>
+                        <hr>
+                        <p>После вашего согласия на подписку вам будут приходить <br>
+                             уведолмения на почту и телефон.</p>
+                        <p>
+                            <label for="username" class="uname" >Ваше имя:</label>
+                            <input id="username" name="username" required type="text" placeholder="Имя"/>
+                        </p>
+                        <p>
+                            <label for="usermail" class="umail" >Email:</label>
+                            <input id="usermail" name="usermail" required type="text" placeholder="email@yandex.ru"/>

```

Comment:
Давай добавим HTML5-валидацию поля для ввода email
```suggestion
                            <input id="usermail" name="usermail" required type="email" placeholder="email@yandex.ru"/>
```